libs/designsystem/accordion/src/accordion.directive.ts

```diff
@@ -1,8 +1,44 @@
-import { Directive } from '@angular/core';
+import {
+  AfterContentInit,
+  ContentChildren,
+  Directive,
+  Input,
+  OnDestroy,
+  QueryList,
+} from '@angular/core';
+
+import { Subscription } from 'rxjs';
+
+import { AccordionItemComponent } from './accordion-item.component';
 
 @Directive({
   selector: '[kirbyAccordion], kirby-accordion',
   exportAs: 'kirby-accordion',
   standalone: false,
 })
-export class AccordionDirective {}
+export class AccordionDirective implements AfterContentInit, OnDestroy {
+  @Input() headingLevel: number;
+  @ContentChildren(AccordionItemComponent) accordionChildren: QueryList<AccordionItemComponent>;
```

Comment:
Consider improving naming a bit to better match how we use content children elsewhere. I know theres one place where we use _children_, but in most places we dont:
```suggestion
  @ContentChildren(AccordionItemComponent) accordionItems: QueryList<AccordionItemComponent>;
```